src/gui/tray/Window.qml

```diff
@@ -569,6 +622,13 @@ Window {
                     sourceSize.width: 64
                 }
 
+                ColorOverlay {

```

Comment:
Just a word of caution, it's not the first ColorOverlay I see used: count how many of those you will have. Are they going to be used in a Component in some Repeater or such? If yes maybe rethink this: they won't come in cheap so expect them to make the rendering (much) slower on some machines.

It might be worth trying to colorize earlier in the pipeline for instance.

Reply:
I already thought about this becoming a problem. However, I'm unsure how to solve it in a more efficient way that is also flexible enough. I'm open to suggestions on how to do that better.